docs/service_upgrade.md

```diff
@@ -0,0 +1,3 @@
+## Changning CoreDNS service type
+Since Kubernetes API doesn't allow to change service type. Helm chart provides default label `k8gb-migrated-svc`,
+when set during helm upgrade, old service deleted and new rolled out.
```

Comment:
It does not provide any instructions anymore. Do we still need this piece of documentation?

Reply:
I'd keep a note about what happens overall

Reply:
Then I guess we should put the statement that it happens fully automatically. Otherwise it is confusing

Reply:
extended a bit